src/test/java/edu/hm/hafner/analysis/assertions/IssueAssert.java

```diff
@@ -0,0 +1,218 @@
+package edu.hm.hafner.analysis.assertions;
+
+import java.util.Objects;
+
+import org.assertj.core.api.AbstractAssert;
+
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.Priority;
+
+/**
+ * Custom Assertion for Issue.
+ *
+ * @author Mark Tripolt
+ */
+public class IssueAssert extends AbstractAssert<IssueAssert, Issue> {
+    /**
+     * Custom constructor.
+     *
+     * @param actual issue
+     */
+    public IssueAssert(final Issue actual) {
+        super(actual, IssueAssert.class);
+    }
+
+    /**
+     * Custom assertThat function for Issue.
+     *
+     * @param actual issue
+     * @return custom assertion
+     */
+    public static IssueAssert assertThat(final Issue actual) {
+        return new IssueAssert(actual);
+    }
+
+    /**
+     * Check for id.
+     *
+     * @param id expected
+     * @return custom assertion
+     */
+    public IssueAssert hasId(final int id) {
+        isNotNull();
+        if (!Objects.equals(actual.getId(), id)) {
+            failWithMessage("Expected Issue Id to be <%s> but was <%s>", id, actual.getId());
```

Comment:
Der Standard ist:
` private static final String EXPECTED_BUT_WAS_MESSAGE = "%nExpecting:%n <%s>%nto be equal to:%n <%s>%nbut was not.";`